### DISCUSSION TITLE
Dashboard: obtain user ID

### DISCUSSION THREAD

**Discussion by bnjmnt4n** [supabase/supabase#2622] [Questions] [answered]
I believe this changed recently, the users page in the dashboard used to show the user ID, which made it very convenient to copy the ID and filter by ID in table or perform a quick SQL query from the dashboard.

Now, to get the user ID, I have to run an additional SQL query, where before I could just copy the IDeasily:

```
select id from auth.users where email = '*'
```

**Comment by silentworks** [comment, marked as answer]
This is still there, you just need to scroll to the right depending on your screen size, its the last column now and not the first like before.

<img width="1124" alt="Screenshot 2021-07-29 at 11 13 16" src="https://user-images.githubusercontent.com/79497/127474574-fafc5218-4167-4854-8096-6674de8f0c63.png">

**Reply by bnjmnt4n** [reply]
It seems like you're right, the issue would then be that there aren't any horizontal scrollbars: I had to zoom out to be able to see the columns which were too wide to fit into my screen.

**Reply by silentworks** [reply]
Ok I've seen what you mean, it's the breakpoint of the responsive table thats causing the issue. I will open a ticket about it and see if we can get it resolved.

<img width="1200" alt="Screenshot 2021-07-29 at 18 07 20" src="https://user-images.githubusercontent.com/79497/127535028-5efae39d-007b-467b-a2f5-c4664d3d54d1.png">

**Reply by silentworks** [reply]
I've now opened an issue on the Issue tracker https://github.com/supabase/supabase/issues/2633
